web/src/locales/en.json

```diff
@@ -295,8 +295,11 @@
   },
   "tooltips": {
     "carbonintensity": "Carbon intensity",
-    "lowcarbon": "Low-carbon",
-    "renewable": "Renewable",
+    "zoneHeader": {
+      "lowcarbon": "Includes all renewables and nuclear.",
+      "renewable": "Electricity sources that can be replenished naturally. These include solar, wind, hydro, biomass and geothermal.",
+      "carbonIntensity": "The CO2 equivalent (CO2eq) measures the global warming impact of greenhouse gases expressed in the equivalent CO2 emissions. gCO2eq/kWh explains how many grams of CO2 equivalent were used to produce one kilowatt-hour of electricity."
```

Comment:
While true there has previously been some confusion as to what this actually means. At the very least we need to include the unit itself, some not everyone would know how to read that annotation otherwise (ie. it would not be clear what -eq means if this is translated).

Reply:
The shorter, the better. I think it works 🤖

Reply:
I think I'll go with this then:
> Grams of CO2 equivalent per kilowatt-hour (gCO2eq/kWh), measuring the greenhouse gas emissions from generating electricity

Just adding to unit to be more explicit.